build.gradle

```diff
@@ -730,10 +739,9 @@ task dockerUpload {
   }
 
   def isInterimBuild = (dockerBuildVersion ==~ /.*-SNAPSHOT/) || (dockerBuildVersion ==~ /.*-alpha/)
-  || (dockerBuildVersion ==~ /.*-beta/) || (dockerBuildVersion ==~ /.*-RC/)
+  || (dockerBuildVersion ==~ /.*-beta/) || (dockerBuildVersion ==~ /.*-RC.*/)
```

Comment:
Is this to support the gitSha suffix?
Is this only going to be used with RC releases?

Reply:
This is to fix an issue with RC naming so it works RC1 etc. and marks them correctly interim builds. Not directly related to the gitsha changes